nixos/roles/ceph/mon.nix

```diff
@@ -19,13 +19,15 @@ in
       supportsContainers = fclib.mkDisableContainerSupport;
 
       primary = lib.mkOption {
+        defaultText = "false";
```

Comment:
Technically not correct as it automatically evaluates to "true" on primary mons, but I am okay with leaving this as Ceph is unused in this release.

Reply:
in most cases it would be useful to add the default value behaviour to the description, especially when it depends on other, more intricate options